ludwig/features/audio_feature.py

```diff
@@ -145,7 +147,11 @@ def _process_in_memory(
         backend,
     ):
         df_engine = backend.df_engine
-        raw_audio = backend.read_binary_files(column, map_fn=read_audio_from_bytes_obj)
+        if version.parse(torch.__version__) > version.parse("2.0.0"):
```

Comment:
Do we want to check for the`torchaudio` version rather than the `torch` version? Is there a case where `torch` is 2.0+ but we would still want to use the else block?

Reply:
I don't recall making this change, so I'm not sure what the specific reasoning of this one is.

Reply:
No worries! Looks like @justinxzhao added this bit, we can revisit it later if necessary.